sia/src/types/currency.rs

```diff
@@ -110,6 +108,10 @@ impl Currency {
         Currency(value)
     }
 
+    pub const fn zero() -> Self {
```

Comment:
You could implement the `Zero` trait https://docs.rs/num/latest/num/traits/trait.Zero.html since you already got `zero` and all that's left is `is_zero`.